litellm/utils.py

```diff
@@ -2431,11 +2431,17 @@ def _map_and_modify_arg(supported_params: dict, provider: str, model: str):
             optional_params["frequencyPenalty"] = {"scale": frequency_penalty}
         if presence_penalty is not None:
             optional_params["presencePenalty"] = {"scale": presence_penalty}
+    
     elif custom_llm_provider == "palm": # https://developers.generativeai.google/tutorials/curl_quickstart
         ## check if unsupported param passed in 
         supported_params = ["temperature", "top_p", "stream", "n", "stop", "max_tokens"]
         _check_valid_arg(supported_params=supported_params)
         
+    elif custom_llm_provider == "gemini": 
+        ## check if unsupported param passed in
+        supported_params = ["temperature", "topK", "topP", "maxOutputTokens", "stopSequences"]
```

Comment:
this should be 
`supported_params = ["temperature", "top_p", "stream", "n", "stop", "max_tokens"]`

the `supported_params` are supported params to litellm.completion https://docs.litellm.ai/docs/completion/input

Reply:
But these are for Gemini API i checked from their Rest API response its same params from there.

Reply:
no but support_params needs to be the args sent to litellm https://docs.litellm.ai/docs/completion/input#input-params-1 

it needs to be one of these params here.